### DISCUSSION TITLE
Welcome to Chatter Discussions!

### DISCUSSION THREAD

**Discussion by privatedev11** [privatedev11/Chatter#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by privatedev11** [comment]
# 📱App Suggestions
Hi all,
If you have a suggestion for an app to be added, make an issue with the app request template.
Thanks,
@privatedev11 

TODO: actually make the issue template

**Comment by privatedev11** [comment]
The updater won't work, bear with me while I find a solution.

**Comment by katastrophe92** [comment]
@privatedev11 Hello. Will you implement mailing providers also? That would be noice. Thanks

**Reply by privatedev11** [reply]
The original idea was to just have chat apps, but I'm sure I can add email to it.

**Reply by katastrophe92** [reply]
That would be awesome. Thanks
